lib/api-helper/goal/fetchGoalsOnCommit.ts

```diff
@@ -35,6 +35,14 @@ import {
 import { goalKeyString } from "./sdmGoal";
 import { goalCorrespondsToSdmGoal } from "./storeGoals";
 
+export function fetchGoalsFromPush(sdmGoal: SdmGoalEvent): SdmGoalEvent[] {
+    const goals = sumSdmGoalEvents((sdmGoal.push as any).goals.filter(g => g.goalSetId === sdmGoal.goalSetId));
```

Comment:
Is `sdmGoal.push` not typed?

Reply:
It is typed to a different type. Internally we are using subscription types but the API is using public `SdmGoalEvent`.